### PR TITLE
Networking with authors, outside an Operation

### PR DESCRIPTION
**Pull request by GabrielMassana**
Pod update needed

### CODE REVIEW

File: Cocoapods/Networking/APIs/Authors/CCPAuthorsAPIManager.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+#import <Foundation/Foundation.h>
+
+#import <CoreNetworking/CoreNetworking.h>
+
+#import "CCPAbstractAPIManager.h"
+
+@interface CCPAuthorsAPIManager : CCPAbstractAPIManager
+
++ (void)retrieveAuthorsWithString:(NSString *)string
```
Comment (wibosco) [inline Cocoapods/Networking/APIs/Authors/CCPAuthorsAPIManager.h]: Not sure about "string", is it being used as a search term or the full author's name? - maybe "searchTerm", "Author" or "Name" for the parameter and "retrieveAuthorsWithName" or "retrieveAuthorsContainingSearchTerm"?

File: Cocoapods/Networking/APIs/Authors/CCPAuthorsAPIManager.h [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                  networkingError:(CCPNetworkingError)networkingError;
```
Comment (wibosco) [inline Cocoapods/Networking/APIs/Authors/CCPAuthorsAPIManager.h]: Do we need a successful block as well?

Comment (GabrielMassana) [inline Cocoapods/Networking/APIs/Authors/CCPAuthorsAPIManager.h]: Maybe to handle no internet connection errors?

**Comment by GabrielMassana** [comment]
The idea of the succes block with the error is to handle no internet connection errors. Or other API errors that we want to show to the user with a nice EasyAlert. :)

**Comment by wibosco** [comment]
@GabrielMassana so the networkingError block is going to be used as a completion where the absence of NSError instance will indicate a successful request?

**Comment by GabrielMassana** [comment]
Yes. Or we can check if we have an error and handle two blocks, 1 for succes and 1 for failure with the error.

**Comment by GabrielMassana** [comment]
I added two block in the last commit. One for success and the other one for failure.

**Comment by wibosco** [comment]
+1 - what i was actually thinking was that we could go down the path where we only one callback block that covers both success and failure (but with more descriptive name than originally proposed)

**Pull request merged**